engine/src/main/java/com/datatorrent/stram/client/ConfigPackage.java

```diff
@@ -37,10 +37,13 @@
 
 import org.apache.commons.io.FileUtils;
 import org.apache.commons.lang3.StringUtils;
+import org.apache.hadoop.conf.Configuration;
 
 import net.lingala.zip4j.core.ZipFile;
 import net.lingala.zip4j.exception.ZipException;
 
+import static com.datatorrent.stram.client.StramClientUtils.jsonFileToAppInfo;
```

Comment:
jsonFileToAppInfo is only used in one place. We should probably just use StramClientUtils.jsonFileToAppInfo, instead of import static here, to be consistent with the rest of StramClientUtils calls.